metadata/licenses/private/rules/license.bzl

```diff
@@ -0,0 +1,65 @@
+"""Declares rule `license`."""
+
+load("//:defs.bzl", "PackageAttributeInfo")
+
+def _license_impl(ctx):
+    metadata = ctx.actions.declare_file("{}.package-metadata.attribute.build.bazel.supply-chain.license.json".format(ctx.attr.name))
+
+    ctx.actions.write(
+        output = metadata,
+        content = json.encode({
+            "kind": "build.bazel.supply-chain.license",
+            "label": str(ctx.label),
+        }),
+    )
+
+    return [
+        DefaultInfo(
+            files = depset(
+                direct = [
+                    metadata,
+                ],
+            ),
+        ),
+        PackageAttributeInfo(
+            metadata = metadata,
+            kind = "build.bazel.supply-chain.license",
```

Comment:
The docs for kind suggest using "reverse DNS format (e.g., `com.example.foo`)". Should this format match the path of the repo containing the attribute? i.e since this is in the bazel-contrib/supply-chain repo, be something like `com.github.bazel-contrib.supply-chain.license`?

Reply:
I was thinking of putting well-known (builtin) attributes under [bazel.build](https://bazel.build) reverse DNS.

I'm still a bit in the fence about the format here: reverse-DNS is simple, but makes it hard to discover the schema of the attributes. We may want to make this a URL to the schema definition (e.g., JSON schema) instead.